tests/schemas/api_v3/regex.js

```diff
@@ -40,6 +40,7 @@ const HEX = /^\b[0-9a-fA-F]+\b$/;
 const TOPIC = /^\b(?:[0-9a-fA-F]+|lsk[a-hjkm-z2-9]{38})\b$/;
 const EMPTY_STRING = /^$/;
 const POS_VALIDATOR_STATUS = /^\b(?:active|standby|banned|punished|ineligible|,)+\b$/;
+const SWAGGER_SCHEMES = /^\b(?:http|https){1}\b$/;
```

Comment:
Can be shortened to, but the existing also looks fine.
```suggestion
const SWAGGER_SCHEMES = /^https?$/;
```